airbyte-integrations/connector-templates/source-python-http-api/acceptance-test-docker.sh

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
 # Build latest connector image
-docker build . -t $(cat acceptance-test-config.yml | grep "connector_image" | head -n 1 | cut -d: -f2-)
+docker build . -t "$(grep "connector_image" acceptance-test-config.yml | cut -d: -f2 | xargs)"
```

Comment:
Unnecessary cat, grep takes a file input already

Reply:
xargs trims all whitespace, which matter now because the entire thing is wrapped in double quotes.

Reply:
> xargs trims all whitespace, which matter now because the entire thing is wrapped in double quotes.

This is useful. Can you add this as a comment in this file? This is the template, so the comment will be preserved for all newly generated source api connectors.